db/migrate/20220915124051_change_user_ref_to_likes.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeUserRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    remove_reference :likes, :user, null: false, foreign_key: true
+    add_reference :likes, :author, null: false, foreign_key: { to_table: 'users' }
+  end
+end
```

Comment:
- [ ] After making the changes please drop the db by running rails `db:drop`, then create it again by running rails `db:create`, and finally make sure that all migrations are successful after running `rails db:migrate`